packages/twenty-server/project.json

```diff
@@ -99,7 +100,7 @@
       "executor": "nx:run-commands",
       "options": {
         "cwd": "packages/twenty-server",
-        "command": "ts-node ../../node_modules/typeorm/cli.js"
+        "command": "ts-node --transpile-only -P tsconfig.json ../../node_modules/typeorm/cli.js"
```

Comment:
Transpile only can be removed if we have a process that runs in parallel during development.

Reply:
But I would try to remove it for the concerned commands instead because it can be tedious to setup a parallel typecheck process that runs in development.

Reply:
@lucasbordeau what would be the case against leaving it to VSCode + CI? 
I'm not very clear on the inner workings but it feels like we are duplicating the work sometime no? I.e. typechecking by VS Code + typechecking by whatever command I launch

Reply:
It's also used in `generate:integration-test` and looks like a generic command that could be used later, so in order to avoid silent behaviors I would at least create a ts-node-no-typecheck command so it's explicit.